Migrate.md

```diff
@@ -0,0 +1,79 @@
+How to migrate from 1.x.x
+=========================
+
+Changes in the namespace
+========================
+
+First off, start by renaming your namespaces, what used to be
+
+```ruby
+class MyClass < Enumeration::Base
+```
+
+will now be
+
+```ruby
+class MyClass < Enumerations::Base
+```
+
+Configuration
+=============
+
+Since v2 has a breaking change of default values in the database columns, you have 2 options:
+
+1. Create an initializer that you will configure to have the old naming conventions
+2. Create data migrations and convert your data to be alligned with the new Enumerations
+
+First option might be tempting, but you need to weigh in the advantages of the new enumerations.
+With the second option, you can get enumerations in the database instead of ID's, which may prove quite usefull.
+
+First way
+=========
+
+You need to create the following initializer
+
+```ruby
+# config/initializers/enumerations.rb
+
+Enumerations.configure do |config|
+  config.primary_key        = :id
+  config.foreign_key_suffix = :id
+end
+```
+
+And voilà! You can carry on with work knowing your enumerations are up to date.
+
+Second way
+==========
+
+Use any gem you see fit, we prefer `data_migrate` gem for this job.
```

Comment:
We do not prefer this or any gem anymore. Just write your migrations as normal but preffering raw sql updates